shex/sio-model/biobank.shex

```diff
@@ -0,0 +1,25 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```

Comment:
we have to use new version in the base URI. Can you please change `v010` to `v020`?